rakelib/rswag.rake

```diff
@@ -63,6 +63,16 @@ namespace :rswag do
       run_tasks_in_parallel(%w[rswag:appeals_api:prod rswag:appeals_api:dev])
     end
   end
+
+  namespace :representation_management do
+    desc 'Generate rswag docs for the representation_management'
```

Comment:
Perhaps change this to `'Generate rswag docs for representation_management'` (i.e. remove the word 'the').

Reply:
Good catch!